wildcards_test.R

```diff
@@ -0,0 +1,147 @@
+# Simple Example with ALL
```

Comment:
Nice sample app! As far as tests go, I imagine we might want to consider
- unit tests in `tests/testthat` (for throwing errors when callbacks are misspecified, after `app$callback` is invoked, but before `app$run_server()` is used)
- a simple integration test to check functionality at adding/removing elements (like with Alex's to-do list app)
- at least some of the checks we implemented in https://github.com/plotly/dash/blob/master/tests/integration/devtools/test_callback_validation.py, we should probably ensure we error out on the same items in R as we do in Python (and maybe provide the same messages when possible)